spec/index.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Spec proposal</title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove"></script>
+    <script class='remove'>
+      // See https://github.com/w3c/respec/wiki/ for how to configure ReSpec
+      var respecConfig = {
+        specStatus: "CG-DRAFT", // FIXME: not a CG-DRAFT, ED?
+        shortName: "lws-ucs",
+        copyrightStart:  "2024",
+        // previousPublishDate:  "N/A",
+        editors: [
+          { name: "Pierre-Antoine Champin" },
+          { name: "Hadrian Zbarcea" },
+        ],
+        group: "solid",  // FIXME: https://respec.org/w3c/groups/ does not list 'lws' yet
+        wg: "Linked Web Storage",
```

Comment:
```suggestion
        github: "w3c/lws-ucs",
```
Not needed, the `wg` option is actually [deprecated](https://respec.org/docs/#wg).
On the other hand, the `github` option adds useful information in the header.